package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s54_basketball_fails",
+  "version": "1.0.0",
+  "description": "## Idea:-  List of Funniest BasketBall Fails",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
```

Comment:
It would be helpful to add some keywords related to the package. This can improve the discoverability of your package. [medium]